mesheryctl/internal/cli/root/constants/constants.go

```diff
@@ -1,9 +1,21 @@
 package constants
 
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/pkg/errors"
+)
+
 var (
 	version        = "Not Set"
 	commitsha      = "Not Set"
 	releasechannel = "Not Set"
+	// the token path includes location to the token for current-context
+	tokenPath = "Not Set"
```

Comment:
pls move the comment on the sameline, for e.g.
 tokenPath     = "Not Set" // comment

Reply:
Sure doing that!